libsolidity/ast/Types.h

```diff
@@ -827,6 +827,8 @@ class StructType: public ReferenceType
 	std::string richIdentifier() const override;
 	bool operator==(Type const& _other) const override;
 	unsigned calldataEncodedSize(bool _padded) const override;
+	// Offset of member in calldata. Does not work for recursive or dynamically encoded types.
+	unsigned calldataOffsetOfMember(std::string const& _name) const;
```

Comment:
Wouldn't this better fit closer to `memoryOffsetOfMember` and `storageOffsetOfMember`?